File: Java/test_app/app/src/main/java/com/example/test_app/MainActivity.java
package com.example.test_app;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.Spinner;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        Button bnt = (Button) findViewById(R.id.button);
        bnt.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Spinner s = (Spinner) findViewById(R.id.spinner);
                TextView t = (TextView) findViewById(R.id.brands);
                t.setText(s.getSelectedItem().toString());
            }
        });
    }
}

File: Kotlin/JetpackCompose/kotlinHelloWorld/app/src/main/java/com/example/kotlinhelloworld/MainActivity.kt
package com.example.kotlinhelloworld

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.TextView
import android.widget.Toast


class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val text: TextView = findViewById(R.id.meu_te)
        // toast show

        val duration = Toast.LENGTH_SHORT
        val toast = Toast.makeText(applicationContext, text.text.toString() , duration)
        toast.show()
    }
}